conf/tol_assembly.config

```diff
@@ -6,22 +6,23 @@ params {
     // "pca,umap,t-sne,isomap,lle_standard,lle_hessian,lle_modified,mds,se,random_trees,kernel_pca,pca_svd,autoencoder_sigmoid,autoencoder_linear,autoencoder_selu,autoencoder_relu,nmf"
 
     // Effectively replaces the Static.config
-    pacbio_barcode_file                 = "${baseDir}/assets/pacbio_adaptors.fa"
+    pacbio_barcode_file                 = "${projectDir}/assets/ascc_database_fasta/pacbio_adaptors.fa"
+    pacbio_barcode_names                = "bc2001,bc2009"
```

Comment:
Should this be set in a production config?

Reply:
I guess not actually, as long as there is a dummy in the nextflow.config.

Reply:
related to comment: https://github.com/sanger-tol/ascc/pull/148/files/3a81f85a8cdddcf33ab6ffc3fac044c1265ccf9a#r2255120033

Reply:
Done